README.md

```diff
@@ -1,18 +1,26 @@
 # OpenBazaar Client v2
 
+This is the refrence client for the OpenBazaar network. It is an interface for your OpenBazaar node, to use it you will need to run an [OpenBazaar node](https://github.com/OpenBazaar/openbazaar-go) either locally or on a remote server.
+
+For full installable versions of the OpenBazaar app, with the server and client bundled together, go to [the OpenBazaar download page.](https://www.openbazaar.org/download/)
+
 [![Build Status](https://travis-ci.org/OpenBazaar/openbazaar-desktop.svg)](https://travis-ci.org/OpenBazaar/openbazaar-desktop)
 
-Installation
-------------
+## Getting Started
+
+To create a local development copy of the reference client, clone the client repository into a directory of your choice:
+- `git clone https://github.com/OpenBazaar/openbazaar-desktop`
+
+Make sure you have the latest versions of Node.js and NPM installed.
```

Comment:
Do we want to give a specific version? Do we know if it will just straight up work with latest node and npm?

Reply:
That's a good point, we should specify the latest versions we've tested.